tests/sql/test_temporal_exprs.py

```diff
@@ -88,6 +91,7 @@ def test_extract():
         extract(date from datetimes) as date,
         extract(day from datetimes) as day,
         extract(dayofweek from datetimes) as day_of_week,
+        extract(dayofyear from datetimes) as day_of_year,
```

Comment:
this is interesting because we have two ways of extracting datetime parts, as well as introducing more non-standard datetime parts. Perhaps we should document what our datetime parts mean now that we have non-standard ones like duckdb?

Reply:
I do agree that we should document the datetime parts for `extract`, but I think that's a bit out of scope for this PR. Let's open up a separate issue for this.

Reply:
https://github.com/Eventual-Inc/Daft/issues/4136